<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewConfigurationWidget</class>
 <widget class="QWidget" name="ViewConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1403</width>
    <height>936</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(208, 208, 208);
background-color: rgb(83, 83, 83);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <widget class="QGroupBox" name="viewListGroupBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>View List</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListView" name="viewListView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addViewBtn">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteViewBtn">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="viewDetailGroupBox">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>View Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="viewNameLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="viewNameLineEdit">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="viewDescriptionLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="viewDescriptionLineEdit">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="bodyPartLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Body Part</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="bodyPartComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="orientationPartLbl_2">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Patient Position</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="patientPositionComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Supine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Prone</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Erect</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lateral</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="projectionProfilePositionLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Projection Profile</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="projectionProfileComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>AP|PA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LAT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OBL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AXIAL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DUAL</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="patientOrientationLbl">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Patient Orientation
(Row,Column)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="patientOrientationRowComboBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>L</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>R</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>P</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>H</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>F</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FP</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="patientOrientationColumnComboBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>L</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>R</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>P</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>H</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>F</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PF</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FP</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="viewPositionLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>View Position</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="viewPositionComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RLD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LLD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RLO</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LLO</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="imageLateralityLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Image Laterality</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="imageLateralityComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>L</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>R</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="imageRotatationLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Image Rotation</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="imageRotationComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>270</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="iconLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Icon</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="iconLineEdit">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="iconSelectFileBtn">
          <property name="text">
           <string>Change</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="collimatorSizeLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Collimator size</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="collimatorSizeXComboBox">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="collimatorSizeYComboBox">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="imageProcessingLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Image Processing</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QComboBox" name="imageProcessComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="labelMarkLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Label Mark</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QComboBox" name="labelMarkComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>R</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>L</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LAT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OBL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FLAT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UPRT</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="labelPositionLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Label position</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QComboBox" name="labelPositionComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LEFT TOP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RIGHT TOP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LEFT BOTTOM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RIGHT BOTTOM</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="positionerProgramLbl">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Positioner Program</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QComboBox" name="positionerProgramComboBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QCheckBox" name="rotateHorizontallyCheckBox">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Rotate horizontally</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
